Source/Core/Core/PowerPC/MMU.cpp

```diff
@@ -1191,6 +1191,9 @@ static void UpdateBATs(BatTable& bat_table, u32 base_spr)
           valid_bit = 0x3;
         else if ((address >> 28) == 0xE && (address < (0xE0000000 + Memory::L1_CACHE_SIZE)))
           valid_bit = 0x3;
+        if (PowerPC::memchecks.OverlapsMemcheck(((batu.BEPI | j) << BAT_INDEX_SHIFT),
```

Comment:
Maybe a small comment explaining why we need this:
`// Fastmem doesn't support memchecks, so disable it for all overlapping virtual pages.`